doc-drafts/cdk-policies.md

```diff
@@ -0,0 +1,60 @@
+<img style="float: left;">
+
+# Policies in CDK
+* Allowlists
+* Denylists
+* AccessControlLists
+
+[Policies](https://docs.polygon.technology/cdk/how-to/manage-policies/#get-information-about-a-policy) in CDK is used for network level access control, and can block specific addresses from entering the pool completely. The use case in Kurtosis may require direct writes to the DB as documented in this guide.
+
+## Note
+* Policies will only "work" when `send_tx` and `deploy` policies are only set to `false`. Setting these policies to `true` will not allow **any** addresses to send transactions at all. 
+* Sometimes it takes longer (> 30 seconds) for the policies to apply, but it seems like there are times where it seems like it never actually gets applied (> 5 minutes), even though the changes to the policies have been made according to the pool_db.
+
+## Blocking addresses
+By default, the two available policies:
+* send_tx
+    * Allows sending txns within the network
+* deploy
+    * Allows deployment of contracts to the network
+
+are set to false. This means the network will not have any restrictions. 
+
+![image](./policies-img/01_policies.png)
+
+Specific addresses can be blocked in this scenario using the below `psql` command:
+
+```
+PGPASSWORD=<zkevm_db_pool_password> psql -h localhost -d pool_db -U pool_user -p <pool_db_postgre_port> -c "INSERT INTO pool.blocked (addr, block_reason) VALUES ('<address_to_block>', NULL) ON CONFLICT DO NOTHING"
+```
+
+After some time (a few seconds with the default Kurtosis configuration) the network will block that address from making entering the pool_db completely.
+
+![image](./policies-img/02_policies.png)
+
+This can be undone by deleting the address from the `pool.blocked` table or using the below `psql` command:
+
+```
+PGPASSWORD=<zkevm_db_pool_password> psql -h localhost -d pool_db -U pool_user -p <pool_db_postgre_port> -c "DELETE FROM pool.blocked WHERE addr = '<address_to_unblock>'"
+
+```
+
+![image](./policies-img/03_policies.png)
+
+## Access Control List (ACL)
+Adding addresses to the ACL will do the opposite of the policy. In this example where `send_tx` and `deploy` are set to `false`, adding addresses to the ACL will block them - similar to adding addresses to the blocklist.
+
+```
+PGPASSWORD=<zkevm_db_pool_password> psql -h localhost -d pool_db -U pool_user -p <pool_db_postgre_port> -c "INSERT INTO pool.acl (address, policy) VALUES ('<address_to_acl>', '<policy>') ON CONFLICT DO NOTHING"
+```
+
+![image](./policies-img/04_policies.png)
+
+```
+PGPASSWORD=<zkevm_db_pool_password> psql -h localhost -d pool_db -U pool_user -p <pool_db_postgre_port> -c "DELETE FROM pool.acl WHERE address = '<address_to_acl>' AND policy = '<policy>') ON CONFLICT DO NOTHING"
+```
+
+## Whitelisting Addresses
```

Comment:
Interesting that this is referred to as allow list earlier and now whitelist? Could be worth while to make sure that nothing got missed due to the naming.

Reply:
It seems like the terms "allowlist" and "whitelist" are interchangeably used here. The docs refer to them as allowlists but the db refers them to as whitelists - I'll clarify this in wording